go/kbfs/libkbfs/subscription_manager.go

```diff
@@ -315,18 +310,127 @@ func (sm *subscriptionManager) SubscribePath(ctx context.Context,
 		ps.debouncedNotify.shutdown()
 		ps.debouncedNotify = sm.makePathSubscriptionDebouncedNotify(ref, limit)
 	}
-	ps.subscriptionIDs[sid] = topic
-	ps.pathsToNotify[path] = struct{}{}
+	ps.subscriptionIDs[req.sid] = req.topic
+	ps.pathsToNotify[req.path] = struct{}{}
 
-	sm.pathSubscriptionIDToRef[sid] = ref
+	sm.pathSubscriptionIDToRef[req.sid] = ref
 	subscriptionIDSetter()
 	return nil
 }
 
+const folderBranchPollingInterval = time.Second
```

Comment:
Once this is merged with the other PR, it would be nice to have a `const` section at the top.